src/screens/ForgotPassword/index.js

```diff
@@ -2,22 +2,39 @@ import React from 'react';
 
 import Alert from 'components/Auth/Alert';
 import Logo from 'components/Auth/Logo';
+import BackButton from 'components/BackButton';
 import PageBackground from 'components/PageBackground';
 import ForgotPasswordForm from 'screens/ForgotPassword/forgotPasswordForm';
+import Handler from 'screens/ForgotPassword/handler';
```

Comment:
```suggestion
import ForgotPasswordHandler from 'screens/ForgotPassword/forgotPasswordHandler';
```

@rafayet-monon Should we keep pattern like what we did before? 🤔
As I see that you use something like this `import ForgotPasswordForm from 'screens/ForgotPassword/forgotPasswordForm';`